src/billo.php

```diff
@@ -32,12 +32,18 @@ function distributeMessage($listsConfig, $mailSender, $id, $message) {
     $mailSender->Subject =
       '['.$listsConfig->getListName($list->getEmail()).'] '.$message->subject;
 
-
+    // Extract and set MIME Header
+    $header = [];
+    $header['Content-Type'] = $message->ContentType;
+    if (isset($message->ContentTransferEncoding)){
+        $header['Content-Transfer-Encoding'] = $message->ContentTransferEncoding;
+    }
+    $mailSender->setBilloHeader($header);
```

Comment:
This is okay, basically what I suggested :-) 

The only nitpick I have now is that we are coming up with a data structure that two parts of the code depend on.  And we are looping twice to generate the array and then to serialize ... maybe we should simply create the string here in place and make the setBilloHeader function even dumber?

With the additional benefit that we could add headers like these multiple times (your array implementation allows only exactly one value per header): 
https://stackoverflow.com/a/18040297/2886832
No idea if we should or need to handle any of these but at least it would give us more flexibility to do so.

Another idea I had was that we could make `getMailMime()` return an empty string and then use `addCustomHeader` but that doesn't allow adding headers with new lines in them, which would require us to fiddle with the header content again which I would like to avoid ... so ... would have been nice but isn't :-D

Reply:
Is it disireable to define a header multiple times? I actually aimed to prevent this XD

Otherwise as suggested in the latest issue, we could just overwrite the `createHeader()` method and simply do what ever we want but this leeds also to more "like to avoid"-stuff.

Reply:
Maybe good enough to use getMailMime() for now